dashboard/app/controllers/sections_controller.rb

```diff
@@ -1,10 +1,20 @@
 class SectionsController < ApplicationController
-  before_action :load_section
+  before_action :load_section_by_code, except: :update
 
   def show
     @secret_pictures = SecretPicture.all.shuffle
   end
 
+  # Allows you to update a section's course_id. Clears any assigned script_id
+  # in the process
+  def update
+    section = Section.find(params[:id])
+    authorize! :manage, section
+
+    section.update!(course_id: params[:course_id], script_id: nil)
+    render json: {}
+  end
```

Comment:
One could argue that this update should happen via section_api_helpers.rb, which is where we make all of our section updates in teacher dashboard. Reasons I chose to do it here instead
(1) Existing code in section_api_helpers only supports wholesale updates (i.e. you need to send all fields) and I want to just PATCH. So wherever this was done would mean new code
(2) The existing code in section_api_helpers better belongs in dashboard and should be moved there at some point
(3) Attempting to make API calls to pegasus from dashboard apparently results in CORS errors.